supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchRuntimeAttributes.scala

```diff
@@ -73,6 +74,7 @@ object AwsBatchRuntimeAttributes {
 
   private val MemoryDefaultValue = "2 GB"
 
+
```

Comment:
Our reviewers tend to like clean diffs, so reverting extraneous changed lines (here) and fixing formatting (above) ahead of time should speed up your review

Reply:
Would like to know the formatting standard that I needs to be adhered to.